julia.html.markdown

```diff
@@ -434,8 +434,8 @@ add(5, 6)
 # => 11
 
 # Compact assignment of functions
-f_add(x, y) = x + y  # => f_add (generic function with 1 method)
-f_add(3, 4)  # => 7
+fAdd(x, y) = x + y  # => fAdd (generic function with 1 method)
+fAdd(3, 4)  # => 7
 
```

Comment:
Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194